### PR TITLE
Changed editor monospace font

### LINKED ISSUE #1026: change editor monospace font to JetBrains Mono
JetBrains Mono is a better font

### PR DESCRIPTION
**Pull request by anirudh-20**
Fixes #1026 

- Editor monospace font has been changed from Ubuntu Mono to JetBrains Mono.
- Style-linting done for changed `.scss` files.

### CODE REVIEW

**Comment by mizlan** [comment]
interesting; was that CSS file never linted before this?

**Comment by anirudh-20** [comment]
> interesting; was that CSS file never linted before this?

I am not sure. Seems like it's been long since it was linted the last time. For this PR, initially the lint checks were failing, so I went ahead and did that.

File: public/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     -->
     <link rel="shortcut icon" href="%PUBLIC_URL%/tla-logo.png">
     <link href="https://fonts.googleapis.com/css?family=Chivo:300,400|Poppins:400,600,700" rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css?family=Anonymous+Pro|Ubuntu+Mono" rel="stylesheet">
```
Comment (mizlan) [inline public/index.html:18]: Was Anonymous Pro being used for anything?

Comment (mizlan) [reply public/index.html:18]: Doesn't look like it; i'll merge

**Review by mizlan — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (anirudh-20) [reply public/index.html:18]: Yes. Anonymous Pro wasn't used anywhere.

**Pull request merged**